charts/pxc-db/templates/cluster.yaml

```diff
@@ -49,25 +53,44 @@ spec:
     apply: {{ .Values.upgradeOptions.apply }}
     schedule: {{ .Values.upgradeOptions.schedule }}
   {{- end }}
+  {{- if .Values.tls }}
+  tls:
+  {{- if hasKey .Values.tls "SANs" }}
+    SANs:
+{{ .Values.tls.SANs | toYaml | indent 6 }}
+  {{- end }}
+  {{- if hasKey .Values.tls "issuerConf" }}
+    issuerConf:
+      name: {{ .Values.tls.issuerConf.name }}
+      kind: {{ .Values.tls.issuerConf.kind }}
+      group: {{ .Values.tls.issuerConf.group }}
+  {{- end }}
+  {{- end }}
+
   {{- $pxc := .Values.pxc }}
   pxc:
     size: {{ $pxc.size }}
     image: "{{ $pxc.image.repository }}:{{ $pxc.image.tag }}"
     autoRecovery: {{ $pxc.autoRecovery }}
+    {{- if $pxc.schedulerName }}
+    schedulerName: {{ $pxc.schedulerName }}
+    {{- end }}
     readinessDelaySec: {{ $pxc.readinessDelaySec }}
     livenessDelaySec: {{ $pxc.livenessDelaySec }}
-    forceUnsafeBootstrap: {{ $pxc.forceUnsafeBootstrap }}
```

Comment:
removed since obsolete?

Reply:
Yes, removed under this ticket: https://jira.percona.com/browse/K8SPXC-831